bin/client

```diff
@@ -51,6 +50,7 @@ lt_client(opt.port, opt, function(err, tunnel) {
     console.log('your url is: %s', tunnel.url);
 
     if (argv.open) {
+        var open_url = require('openurl');
```

Comment:
please add a comment above this explaining why this require is here otherwise future change might break this.

Another alternative could be to keep the require at the top of the file and wrap it in a `try/catch` (still with a comment explaining why).